models/grading/fact_student_status.sql

```diff
@@ -5,6 +5,8 @@ select
     courses.course_name as course_name,
     courses.course_run as course_run,
     coalesce(state, 'failed') as state,
+    enrollment_status,
+    lg.emission_time as emission_time,
```

Comment:
In the context of this model, what is `emission_time` supposed to represent? Since this is combining events from several different places, does it make sense to include the timestamp from only one of those events?

Reply:
I think @daniriosp has a better answer for this than I do. I think it makes sense to remove the emission time to avoid confussion.